src/App.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Landing from './components/landing/Landing'
 
 const App: React.FC = () => {
   return (
-    <div className="App">
-      <h1>High tech React app NOW WITH TYPESCRIPT!!!!!</h1>
-      <h1>Please create your own branch before working with this project!</h1>
-      <h1>And please put your components in the src/components folder.</h1>
-    </div>
+    <Router>
+      <div className="App">
+        <Landing></Landing>
```

Comment:
```suggestion
        <Landing />
```
Don't remove the indentation